src/main/kotlin/com/hibob/academy/dao/OwnerDao.kt

```diff
@@ -0,0 +1,36 @@
+package com.hibob.academy.dao
+
+import jakarta.inject.Inject
+import org.jooq.DSLContext
+import org.jooq.Record
+import org.jooq.RecordMapper
+import kotlin.random.Random
+
+class OwnerDao @Inject constructor(private val sql: DSLContext) {
+
+    private val owner = OwnerTable.instance
+    private val companyId = Random.nextLong()
```

Comment:
Why is the companyId a random long? we want to get explicitly the companyId as parameter in the functions